dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/ExportSecretForm.java

```diff
@@ -8,15 +9,14 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
-import javax.validation.constraints.Max;
-import javax.validation.constraints.Min;
-import javax.validation.constraints.NotNull;
+import com.dotcms.repackage.javax.validation.constraints.Max;
+import com.dotcms.repackage.javax.validation.constraints.Min;
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
 
 public class ExportSecretForm extends Validated {
 
     @NotNull
-    @Min(14)
-    @Max(32)
+    @Size(min = 14, max=32, message="The password must be a string with a length of 14 and 32")
```

Comment:
"with a length between 14 and 32"

Reply:
done https://github.com/dotCMS/core/pull/21456/commits/c6b66f24fc7828606ff2281713c2d2d552d550fb